### ISSUE TITLE
vb5run fails to install with return status 5

### ISSUE THREAD

**Issue by DOOMMARINE117** [Winetricks/winetricks#2255]
here is my issue. 

```
Executing load_vb5run 
grep: warning: stray \ before /
Executing cd /home/talion117/.cache/winetricks/vb5run
Executing wine msvbvm50.exe
02fc:fixme:setupapi:SetupDefaultQueueCallbackW notification 262144 params 31f58c,0
------------------------------------------------------
warning: Note: command wine msvbvm50.exe returned status 5. Aborting.
```

i tried 64 bit or 32 bit version. no luck. any ideas?

**Comment by DOOMMARINE117** [comment]
nevermind. seems i was able to fix it.

**Comment by DOOMMARINE117** [comment]
now im thinking about

**Comment by DOOMMARINE117** [comment]
yup, still there. i thought i had it fixed,. but nope. still there for vb5run and vb65un. 

i got confused on it.

**Comment by Chiitoo** [comment]
I do see the issue with `vb5run`, but that is most likely a Wine issue (https://bugs.winehq.org/).

However, I don't see errors with `vb6run` here.  Please post the complete output from trying to install that.

**Comment by DOOMMARINE117** [comment]
> I do see the issue with `vb5run`, but that is most likely a Wine issue (https://bugs.winehq.org/).
> 
> However, I don't see errors with `vb6run` here. Please post the complete output from trying to install that.

where do i find the full outpost of it?

**Comment by Chiitoo** [comment]
You can open a terminal emulator window, and run the command there (`winetricks vb6run`), then save the output to a file and attach the file here, or copy/paste from that window directly.

**Comment by DOOMMARINE117** [comment]
winetricks vb5run
------------------------------------------------------
warning: You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Using winetricks 20240105 - sha256sum: 8f28985b06e9e04b5268e2aef3429b2a0f6abd4fd12c4a30472dfe66355128fa with wine-9.14 and WINEARCH=win64
Executing w_do_call vb5run
------------------------------------------------------
warning: You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Executing load_vb5run 
grep: warning: stray \ before /
Executing cd /home/talion117/.cache/winetricks/vb5run
Executing wine msvbvm50.exe
0248:fixme:setupapi:SetupDefaultQueueCallbackW notification 262144 params 42f58c,0
------------------------------------------------------
warning: Note: command wine msvbvm50.exe returned status 5. Aborting.

**Comment by Chiitoo** [comment]
Yep, that's what I confirmed for `vb5run`, but what do you get for `vb6run`?

**Comment by DOOMMARINE117** [comment]
i was able to get vb6run somehow. it seems to be only vb5run now.

**Comment by DOOMMARINE117** [comment]
so what should i do for vb5run and directshow 64 bit version?

**Comment by Chiitoo** [comment]
> so what should i do for vb5run

Is `vb6run` not enough for your purposes?  In any case, as I mentioned, the issue with that seems to be most likely happening due to a bug in Wine, and as such, it should be reported via the Wine bug tracker: https://bugs.winehq.org/

> directshow 64 bit version?

`winetricks directshow`

**Comment by austin987** [comment]
https://bugs.winehq.org/show_bug.cgi?id=56209

**Comment by Chiitoo** [comment]
Ah, it is already up there.  :]

Will close this now since there is nothing we can do about it here.

If there are issues with `winetricks directshow`, do open a new issue about that.

Thank you!

**Comment by Chiitoo** [comment]
Sorry, maybe it is better to keep this one open until it is fixed upstream, or removed from Winetricks.
